### ISSUE TITLE
Tip.html - Object #<klass> has no method 'trigger'

### ISSUE THREAD

**Issue by mattpowell** [Droppe/Lu#31]
Steps to repro:
1) Checkout latest AthenaUIFramework
2) Start server: node server.js
3) In Chrome, open: http://localhost:3000/examples/Tip.html

Stacktrace:

```
Uncaught TypeError: Object #<klass> has no method 'trigger'
Athena.execute.setData.executed                                         /scripts/libraries/athena/athena.js:194
_.each._.forEach                                                                       underscore.js:82
execute                                                                                        /scripts/libraries/athena/athena.js:193
Athena.execute                                                                          /scripts/libraries/athena/athena.js:240
jQuery.extend.each                                                                   jquery-1.7.js:660
jQuery.fn.jQuery.each                                                               jquery-1.7.js:274
Athena.execute                                                                          /scripts/libraries/athena/athena.js:237
require.ensure.ensureExecutionCallback                            inject.js:1279
execute                                                                                        inject.js:1015
afterDownload                                                                            inject.js:977
afterDownload                                                                            inject.js:977
afterDownload                                                                            inject.js:977
afterDownload                                                                            inject.js:977
```

Also, Tip:Right seems to still work...

**Comment by mattpowell** [comment]
Seems to stem from athena.js, line 176...

**Comment by Jimgskoop** [comment]
Tip or Tab?

-Jim G

On Dec 19, 2011, at 9:40 AM, Matt Powell <reply@reply.github.com

**Comment by mattpowell** [comment]
Whoops, sorry... it's Tip.html. It's still early ;)

**Comment by mattpowell** [comment]
Resolved w/ this commit: 13c7b3052a
